src/xdocs/config_javadoc.xml

```diff
@@ -64,7 +64,8 @@
               <td>
                 <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#CLASS_DEF">
                   CLASS_DEF</a>,
-                <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#INTERFACE_DEF">
+                <a href=
+                   "apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#INTERFACE_DEF">
                   INTERFACE_DEF</a>,
```

Comment:
> An UT has been provided to enforce this formatting
    source:
    #3946 (comment)
> 
> TOKEN1,
TOKEN2,
TOKEN3.
> 
> to:
> 
> TOKEN1
,TOKEN2
,TOKEN3
.

UT is not doing what is described in first post of PR or in issue.

Reply:
This is not a tokens or javadocTokens property, but I believe we are still validating its contents as part of the same test.

Reply:
Done, for all such occurences